index.js

```diff
@@ -0,0 +1,10 @@
+const hamburger = document.querySelector('.menu-button');
+const menu = document.querySelector('.drop-menu');
+const logo = document.querySelector('.logo');
+const menuButton = document.querySelector('.menu-button');
+
+hamburger.addEventListener('click', () => {
+  menu.classList.toggle('active');
+  logo.classList.toggle('active');
+  menuButton.classList.toggle('active');
+});
```

Comment:
- [ ] Kindly make sure that when the user clicks any option of the mobile menu, the mobile menu should disappear. As you can see in the attached video that the mobile menu is persistent and doesn't disappear without clicking the cross (X) button.

https://user-images.githubusercontent.com/70269507/161868705-55d8d2e6-812b-4480-a6a1-fe78aa2d5873.mov

Reply:
Thank you